src/schema/rules/files/raw/task.yaml

```diff
@@ -30,6 +30,7 @@ timeseries:
     - eeg
     - ieeg
     - nirs
+    - motion
```

Comment:
that's not going to work because the general rule for time series does not contain the tracksys required entity for motion

So the rendered filename template do not contain it: https://bids-specification--1458.org.readthedocs.build/en/1458/modality-specific-files/motion.html#motion-recording-data

so you need to add a new section in the yaml file with this:

```
timeseries__motion:
  $ref: rules.files.raw.task.timeseries
  datatypes:
    - motion
  entities:
    $ref: rules.files.raw.task.timeseries.entities
    tracksys: required
```